TCMManagement/BusinessLayer/AppointmentService.cs

```diff
@@ -81,7 +81,7 @@ public IEnumerable<Appointment> GetItems(IEnumerable<KeyValuePair<string, string
                     }
                 }
             }
-            return (new List<Appointment>());

```

Comment:
Why we return all rows in table if the query request is blank ? I see this is consistent in all service classes. Just wonder why we handle the blank query request like this.

Reply:
Oh.
I just found that it will be easier for the front-end development.

Reply:
OK, get it.